CHANGELOG.md

```diff
@@ -6,6 +6,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [15.0.0]
+### Changed
+- **BREAKING** Update TokensController allTokens state structure to make network/chainID parent of account ([#572](https://github.com/MetaMask/controllers/pull/572))
+  - The shape of allTokens state field on TokensController has been reorganized. Consumers of the TokensController will have to migrate existing token state to new shape.
+- **BREAKING** ignoredTokens changed to allIgnoredTokens ([#570](https://github.com/MetaMask/controllers/pull/570))
+  - a new state field on the TokensController - allIgnoredTokens - now manages ignoredTokens by network and accountAddress, ignoredTokens is now the array of token address strings (previously an array of full token objects) that have been hidden by the user for the currently active network and accountAddress pair. Consumers of the TokensController will have to migrate existing ignoredTokens array to allIgnoredTokens object.
+- **BREAKING** Improve BaseControllerV2 messenger type ([#556](https://github.com/MetaMask/controllers/pull/556))
+  - This is a breaking change, because anyone extending BaseControllerV2 will now be required to supply an additional generic parameter.
+- **BREAKING** Remove redundant default export from util.ts ([#574](https://github.com/MetaMask/controllers/pull/574))  
+  - This is potentially breaking for consumers who import methods from the utils.ts as default/unnamed imports and will require changing these to named imports.
+- **BREAKING** Removing aggregator from TokenListToken ([#564](https://github.com/MetaMask/controllers/pull/564))
+  - This is breaking because the the DynamicToken and TokenListToken types no longer contain an aggregators field. Consumers will have to remove aggregators for objects using this type.
+- **BREAKING** Migrate ApprovalController to BaseControllerV2 ([#555](https://github.com/MetaMask/controllers/pull/555))
+  - This is a breaking change because the 'resolve' method has been renamed to 'accept'.
```

Comment:
The migration to BaseControllerV2 is a breaking change all on its own, even if this method wasn't renamed. Config is gone and can't be changed at runtime, the constructor is different, the way you set state is different, etc. Too many changes to list here.

I don't love this suggestion but this is the best wording I could come up with:
```suggestion
  - This is a breaking change because the BaseControllerV2 migration is breaking, and the 'resolve' method has been renamed to 'accept'.
```